teaspoon.gemspec

```diff
@@ -20,4 +20,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.4"
   s.add_dependency "railties", ">= 3.2.5"
+  s.add_dependency "capybara"
+  s.add_dependency "selenium-webdriver"
+  s.add_dependency "webdrivers"
```

Comment:
these are already over here: https://github.com/jejacks0n/teaspoon/blob/master/teaspoon-devkit.gemspec

Reply:
and they shouldn't be dependencies of teaspoon, because you might not use them, unless they are now? haha